common/djangoapps/student/views.py

```diff
@@ -1093,6 +1104,42 @@ def change_enrollment(request, check_access=True):
         return HttpResponseBadRequest(_("Enrollment action is invalid"))
 
 
+@transaction.non_atomic_requests
+@require_POST
+@outer_atomic(read_committed=True)
+def update_subscription(request):
+    """
+    Modify the subscription_until date.
+
+    Args:
+        request (`Request`): The Django request object
+
+    Returns:
+        Response
+
+    """
+    # Get the user
+    user = request.user
+
+    # Ensure the user is authenticated
+    if not user.is_authenticated():
+        return HttpResponseForbidden()
+
+    order_date = request.POST.get("order_date")
+    order_status = request.POST.get("order_status")
+    if order_status == 'Complete':
```

Comment:
You might want to import `COMPLETE` from `lms/commerce/constants.py` if this is the same constant :)